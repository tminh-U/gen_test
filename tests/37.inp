6
13 12 1 17 3 7 
14
3 4
3 5
1 4
1 2
1 4
5 5
1 4
6 6
6 6
2 4
3 6
5 6
2 6
4 6
