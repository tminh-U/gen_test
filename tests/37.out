4
6
8
3
8
2
8
2
2
6
10
4
12
5
